content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -130,31 +87,33 @@ can set the number with `--jobs` (`-j`).
 If an experiment being pulled already exists in the local project, DVC won't
 overwrite it unless you supply `--force`.
 
-### Example: Pushing or pulling multiple experiments
+## Example: Sharing multiple experiments
 
-You can create a loop to upload or download all experiments like this:
+You can create a loop to push or pull all experiments. For example in a Linux
```

Comment:
This is not example - this is How to. Not clear again what is the motivation for this "example"

Reply:
Suppose you have thousands of experiments, would you push/pull/share them one by one?

Reply:
Agree it's more of a how to... Moved to a new how-to page for now. PTAL

I also agree it's a clear need @iesahin - maybe we should open a feature request in the core repo though? Maybe `exp push/pull --head` or/and `<branch>` or/and `--all`